web/src/pages/HomePage/style.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../theme-variables.scss";
+
+.home-page {
+    height: 100vh;
+    background: linear-gradient(-45deg, #000 0%, #052892 100%);
```

Comment:
Let's change the colour to Y-Check colours.
You can also do your own design if you prefer.

Reply:
![Uploading image.png…]()

Reply:
<img width="1117" alt="image" src="https://github.com/dodziraynard/y-check-ug/assets/81252456/cfc2fe61-f6fc-46dd-840a-94cb9a8a8ca4">